lib/util/fs/index.spec.ts

```diff
@@ -1,3 +1,4 @@
+import _findUp from 'find-up';
```

Comment:
```suggestion
import * as _findUp from 'find-up';
```
You probably need to import as namespace.

Reply:
@viceice I've tried to do as you suggested, but TypeScript still complaints about `mockResolvedValueOnce()` missing. I guess this is related to `find-up`, since this module looks rather special. I can try updating its version from current to latest `6.3.0` and see if that helps, what do you think?